pay-api/src/pay_api/utils/converter.py

```diff
@@ -95,6 +95,8 @@ def remove_nones(data: Dict[Any, Any]) -> Dict[str, Any]:
         for key, val in data.items():
             if isinstance(val, dict):
                 new_data[key] = Converter.remove_nones(val)
+            elif isinstance(val, list):
+                new_data[key] = [Converter.remove_nones(item) if isinstance(item, dict) else item for item in val]
```

Comment:
this can impact performance and effect other fields I think we were just getting rid of None at the invoice base level, but PLI for example needed this extra bit